### PR TITLE
kue: Put kue behind a feature flag.

### PR DESCRIPTION
**Pull request by j0j00**
- DEV UI server: Persist the previous season number and episode so you don't have to retype it when you retry.

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -13,4 +13,6 @@ LOG_LEVEL="debug"
 # Winston logs
 SILENCE_LOGS="false"
 # Winston file logging
-FILE_LOGGING="false"
\ No newline at end of file
+FILE_LOGGING="false"
+# Kue queue processing
+CLAWS_KUE=
```
Comment (SpajicM) [inline .env.dist]: default value?

Comment (j0j00) [reply .env.dist]: A value is not required, it's the same as an empty string.

Comment (j0j00) [reply .env.dist]: A value is not required, it's the same as an empty string.

**Comment by ddivad** [comment]
@j0j00 thanks for the pr.

I might cherry pick these changes into my Kue branch later. This is because I have other changes related to the kue stuff that I want to add in my branch before merging Kue to master.

**Comment by ddivad** [comment]
Closing as these changes will be added in my Kue PR.

**Pull request closed without merging**